packages/design-system/src/components/TimePicker/index.tsx

```diff
@@ -143,23 +144,25 @@ export const TimePicker: FC<TimePickerProps> = ({
           stylesInputVariant={stylesInputVariant}
         />
       )}
-      <TimePickerDropdown
-        isOpen={isDropdownOpen}
-        hoursFormat={hoursFormat}
-        minutesFormat={minutesFormat}
-        secondsFormat={secondsFormat}
-        timeFormat={timeFormat}
-        setTime={setTime}
-        time={time}
-        handleOKClick={handleOKClick}
-        handleCancelClick={handleCancelClick}
-        inputRef={inputRef}
-        withPeriodButtons={withPeriodButtons}
-        className={dropdownClassName}
-        hoursStartScrollIndex={hoursStartScrollIndex}
-        minutesStartScrollIndex={minutesStartScrollIndex}
-        secondsStartScrollIndex={secondsStartScrollIndex}
-      />
+      <div className={classNames("absolute", !!label ? "top-6" : "top-0")}>
```

Comment:
```suggestion
      <div className={classNames("absolute", label ? "top-6" : "top-0")}>
```
You don't have to use `!!label`.